.golangci.yml

```diff
@@ -150,6 +150,9 @@ issues:
     - text: "Expect WriteFile permissions to be 0600 or less"
       linters:
         - gosec
+    - text: 'shadow: declaration of "(err|ctx)" shadows declaration at'
```

Comment:
Could you please show me some context for the ctx usage within our codebase? If not, I argue its premature to introduce it. Wait for the PR where you use it and we can judge it on its merits then

Reply:
I'll remove.
Duplicate discussion and reply [here](https://github.com/thrasher-corp/gocryptotrader/pull/1754/files#r1899248775)

Fixed https://github.com/gbjk/gocryptotrader/commit/c19f66c5a

Folded down immediately into https://github.com/gbjk/gocr yptotrader/commit/f21d44717 so I can rename the commit.